fix various delegate docs (#635)
harness/developer-hub: docs/platform/2_Delegates/secure-delegates/trust-store-override-for-delegates.md

Harness Delegates perform most Harness tasks. Delegates make outbound TLS/SSL connections to the Harness SaaS platform to obtain these task assignments. The TLS/SSL connection from the delegate to Harness requires a trusted certificate.

> Harness Delegate ships with a Java Runtime Environment (JRE) that includes a default trusted certificate in its [truststore](https://docs.oracle.com/cd/E19830-01/819-4712/ablqw/index.html) (located at `/etc/pki/java/cacerts`). This truststore uses multiple trusted certificates, however, you might want to limit them to conform to your company's security protocols.

Reviewer: ```suggestion
Harness Delegate ships with a Java Runtime Environment (JRE) that includes a default trusted certificate in its [truststore](https://docs.oracle.com/cd/E19830-01/819-4712/ablqw/index.html) (located at `/etc/pki/java/cacerts`). This truststore uses multiple trusted certificates. You can limit the number you use based on your company security protocols.
```

> Harness' only requirement is that the JRE truststore includes the certificate delegates use to trust Harness (app.harness.io).

Reviewer: ```suggestion
The JRE truststore must include the certificate that delegates require to establish trust with Harness (app.harness.io).
```

Open the delegate YAML file in a text editor.

> In the `Deployment` manifest, under `env`, locate `JAVA_OPTS`.

Reviewer: ```suggestion
In the `Deployment` specification, locate the `env` field, and then find the `JAVA_OPTS` environment variable.
```
